backend/scripts/debug_ai_processing.py

```
@@ -0,0 +1,316 @@
+#!/usr/bin/env python3
+"""
+AI处理调试脚本
+用于排查 summary 和 key_points 处理问题
+"""
+
+import asyncio
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)